sormas-api/src/main/java/de/symeda/sormas/api/common/Page.java

```diff
@@ -0,0 +1,68 @@
+package de.symeda.sormas.api.common;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class Page<T> implements Serializable {
+    List<T> elements;
+    int pageNumber = 0;
+    int size = 50;
+    long totalNoElements=0;
```

Comment:
I guess this is short for "total number of elements"? Maybe "totalElementCount" would be a bit clearer (because "NoElements" could be mistaken for "no elements", theoretically)?